redis/client.py

```diff
@@ -959,11 +959,12 @@ def expireat(self, name, when):
             when = int(mod_time.mktime(when.timetuple()))
         return self.execute_command('EXPIREAT', name, when)
 
-    def get(self, name):
+    def get(self, name, default=None):
         """
         Return the value at key ``name``, or None if the key doesn't exist
         """
-        return self.execute_command('GET', name)
+        return_value = self.execute_command('GET', name)
+        return [return_value, default][return_value is None]
```

Comment:
because `[], {}, ''` etc. would also yield `True`, so for empty key it could return the default instead of empty string